go.mod

```diff
@@ -233,7 +233,7 @@ replace (
 	github.com/coreos/go-oidc => github.com/coreos/go-oidc v2.1.0+incompatible
 	github.com/coreos/go-semver => github.com/coreos/go-semver v0.3.0
 	github.com/coreos/go-systemd => github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
-	github.com/coreos/pkg => github.com/coreos/pkg v0.0.0-20180108230652-97fdf19511ea
+	github.com/coreos/pkg => github.com/coreos/pkg v0.0.0-20160727233714-3ac0863d7acf // 3ac0863d7acf is the SHA for git tag v3
```

Comment:
we should use the `v4` tag which matches what we're already pointing to (https://github.com/coreos/pkg/commits/v4)

Reply:
Done!